raiden/network/transport/matrix/transport.py

```diff
@@ -494,7 +495,7 @@ def start(  # type: ignore
 
         self.log = log.bind(
             current_user=self._user_id,
-            node=self.checksummed_address,
+            node=to_checksum_address(self._raiden_service.address),
```

Comment:
`RaidenService.address` is a `bytes` address...

Reply:
There is (or was) a property called `checksummed_address` which did exactly the line below. But I removed it again as it does not seem to be used at all.